Python/tests/package_management_pypi_test.py

```diff
@@ -164,62 +227,22 @@ def check_version():
     finally:
         _drop_all_ddl_packages(connection, scope)
 
-
-def test_install_abslpy():

```

Comment:
why are we removing these tests? Is the scenario covered by other test?

Reply:
The Theano test is moved up in the file.

The absl test functionality is covered by previous tests. It is not unique except that it tests the absl-py package, which is also pretty slow. Since this is for the CI pipeline, I wanted to remove the redundant and slow tests.